### ISSUE TITLE
Keep getting an error with SmoothStarRating. Please help. It is urgent

### ISSUE THREAD

**Issue by 5ran6** [inspireui/support#3206]
From the first time I ran till now (over 24 hours after purchasing the code), I keep getting this same error. I have tried all I can do. Please help. The error is:

Compiler message:
/C:/flutter/.pub-cache/hosted/pub.dartlang.org/rate_my_app-0.6.0+2/lib/src/dialogs.dart:272:17: Error: No named parameter with the name 'onRatingChanged'.
                onRatingChanged: (rating) {
                ^^^^^^^^^^^^^^^
/C:/flutter/.pub-cache/hosted/pub.dartlang.org/smooth_star_rating-1.1.0+1/lib/smooth_star_rating.dart:23:3: Context: Found this candidate, but the arguments don't match.
  SmoothStarRating({
  ^^^^^^^^^^^^^^^^
Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
build failed.

FAILURE: Build failed with an exception.

* Where:
Script 'C:\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 780

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 16s
Finished with error: Gradle task assembleDebug failed with exit code 1

**Comment by minhcasi** [comment]
Thank you for the feedback.

Please help us to submit this form and fill all required info to continue with the support  - http://private.inspireui.com/ - please don't share the purchase code or screenshot public here as another user will use it for activation.

We will close this ticket in 24 hours if it is not provided enough detail, kindly reopen the ticket after you have done the form submission. Thank you for your time!

**Comment by Alnaggar94** [comment]
change rate_my_app: 0.6.0+2 to rate_my_app: 0.6.0+3 / in pubspec.yaml

**Comment by ameenbadri7** [comment]
before we can fix others issue. we you should know whether they have submitted the purchase code or not @Alnaggar94 . thank you very much my man. 

>

**Comment by minhcasi** [comment]
@5ran6 @ameenbadri7 

You could also try to update latest `rate_my_app: 0.6.1` to fix this issue.

Kindly help to create a new ticket if you any issue, also submit with the verify purchase to http://private.inspireui.com - thank you!
